Tools/machines/lxplus-cern/spack.yaml

```diff
@@ -0,0 +1,62 @@
+spack:
+  # variations of this environment can be controlled via environment variables during `spack env create`
+  definitions:
+  # CUDA
+  - cuda: []
+  - cuda: [cuda, blaspp +cuda]
+    when: env.get("SPACK_STACK_USE_CUDA", "") == "1"
+
+  # basic Python dependencies for WarpX Python PICMI bindings
+  - python: []
+  - python: [python, py-pip]
```

Comment:
Do be discussed:
to simplify things, we could just install Python and/or CUDA by default and spare the variations logic :)